docs/en/integrations/data-ingestion/google-dataflow/dataflow.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_label: Integrating Dataflow with ClickHouse
+slug: /en/integrations/google-dataflow/dataflow
+sidebar_position: 1
+description: Users can ingest data into ClickHouse using Google Dataflow
+---
+
+# Integrating Google Dataflow with ClickHouse
+
+[Google Dataflow](https://cloud.google.com/dataflow) is a fully managed stream and batch data processing service. It supports pipelines written in Java or Python and is built on the Apache Beam SDK.
+
+There are two main ways to use Google Dataflow with ClickHouse, both are leveraging [`ClickHouseIO`](../../apache-beam):
```

Comment:
@gingerwizard do we have any guidelines on using relative vs absolute links?

Reply:
@mshustov we should prefer relative links to files. Doing so will help the framework resolve versions and translations. Source: https://docusaurus.io/docs/markdown-features/links